engine/evarutil.mli

```diff
@@ -126,13 +125,18 @@ val gather_dependent_evars : evar_map -> Evar.t list -> (Evar.Set.t option) Evar
     solved. *)
 val advance : evar_map -> Evar.t -> Evar.t option
 
+(** [reachable_from_evars sigma seeds ev] computes if [ev] is a descendent
+    of an evar of [seeds] by restriction or evar-evar unifications in [sigma]. *)
+val reachable_from_evars : evar_map -> Evar.Set.t -> Evar.t -> bool
```

Comment:
During the WG I was suggesting the type `evar_map -> Evar.Set.t -> Evar.Set.t`.
At least, this is the function I'd like to have for implement some features of ssr.

Reply:
Thanks for the reminder. The semantics should be that any one of the evars in the set is reachable, right?